server.js

```diff
@@ -6,6 +6,15 @@ import connectQueue from './utils/connectQueue.js'
 import isMerged from './utils/isMerged.js'
 dotEnv.config()
 
+const jobOptions = {
```

Comment:
mending di simpen di utils aja supaya setiap code confifg semua ada di folder yg sama,, kaya koneksi koneksi db, token aplikasi 3rd, dsb

Reply:
in here dude @sandisunandar99 :+1: 
9a13115

Reply:
okayy